build.sbt

```diff
@@ -422,6 +422,22 @@ lazy val reflect = configureAsSubproject(project)
   )
   .dependsOn(library)
 
+lazy val exporter = configureAsSubproject(project)
```

Comment:
A more descriptive name and/or comment explaining what this project does might be nice